index.bs

```diff
@@ -546,7 +546,9 @@ To <dfn>initialize the session</dfn>, given |session|, |mode|, and |device|, the
   1. Set |session|'s [=XRSession/mode=] to |mode|.
   1. Set |session|'s [=XRSession/XR device=] to |device|.
   1. [=Initialize the render state=].
-  1. If no other features of the user agent have done so already, perform the necessary platform-specific steps to initialize the device's tracking and rendering capabilities.
+  1. If no other features of the user agent have done so already, perform the necessary platform-specific steps to initialize the device's tracking and rendering capabilities, including showing any necessary instructions to the user.
+
+NOTE: Some devices require additional user instructions for activation, for example going into immersive mode on a phone-based headset device requires inserting the phone into the headset, and doing so on a desktop browser connected to an external headset requires wearing the headset. It is the responsibility of the user agent &mdash; not the author &mdash; to show any such instructions.
```

Comment:
NOTE: => Note:

Also, minor wording quibble:

> It is the responsibility of the user agent &mdash; not the author &mdash; to show any such instructions.

Even though this text is non-normative, we don't want to imply that the UA must be the one showing the instructions instead of delegating that to the platform when appropriate. (IE: For Daydream it's the Daydream service, not Chrome, that shows the harness instructions.) Perhaps it would be more accurate to say:

> It is the responsibility of the user agent &mdash; not the author &mdash; to **ensure any such instructions are shown**.

Regardless, the most important part is simply that content authors shouldn't be trying to show instructions and you've captured that well.

Reply:
> NOTE: => Note:

fwiw both work in bikeshed syntax

Reply:
Resolved.

Reply:
> fwiw both work in bikeshed syntax

Yup, and we've used both intermixed in the past. I did a pass over the whole doc to normalize it recently, though, and so this is just to keep with style conventions.